src/nyc_trees/apps/core/models.py

```diff
@@ -136,6 +136,10 @@ def eligible_to_become_trusted_mapper(self, group):
             not user_is_trusted_mapper(self, group)
 
 
+def _generate_image_filename(group, filename):
+    return 'group_images/{}/{}'.format(group.slug, filename)
```

Comment:
`AWS_S3_FILE_OVERWRITE` is `True` by default, which seems to mean that similar file paths would overwrite each other. It might be good to think through the scenarios for generating the paths on S3 to ensure that multiple users can't overwrite each others files.